apps/schematic/api/schematic_api/controllers/storage_controller_impl.py

```diff
@@ -159,6 +159,56 @@ def get_dataset_files(
     return result, status
 
 
+def get_dataset_manifest_from_schematic(
+    asset_type: str, dataset_id: str
+) -> pd.DataFrame:
+    """Gets a manifest in pandas.Datframe format
+
+    Args:
+        asset_type (str): The type of asset, ie "synapse"
+        manifest_id (str): The unique id for the manifest file
+        dataset_id (str): The id of the dataset the manifest is in
+
+    Returns:
+        pandas.DataFrame: The manifest
+    """
+    access_token = get_access_token()
+    asset_type_object = get_asset_storage_class(asset_type)
+    store = asset_type_object(access_token=access_token)
+    manifest_data = store.getDatasetManifest(
+        datasetId=dataset_id, downloadFile=True, newManifestName="manifest.csv"
+    )
+
+    manifest_local_file_path = manifest_data["path"]
+    manifest = pd.read_csv(manifest_local_file_path)
```

Comment:
Instead of using `pd.read_csv` we might want to use the schematic function for reading in CSVs to maintain a unified approach (`schematic.utils.df_utils.load_df`)

Reply:
Should this opther endpoint do the same?

Reply:
@andrewelamb yes, sorry I missed that then